libr/bin/format/elf/elf.c

```diff
@@ -4267,6 +4276,22 @@ static bool _add_sections_from_phdr(RBinFile *bf, ELFOBJ *eo, bool *found_load)
 	}
 
 	int i = 0, n = 0;
+
+	// Calculate base address delta for sBPF with custom baddr
+	st64 base_delta = 0;
```

Comment:
uhm shouldnt this be implemented in the get_baddr instead?

Reply:
not really, this is relocating the sections vaddr relative to the new baddr